api/src/main/java/org/openmrs/module/m2sysbiometrics/M2SysBiometricsConstants.java

```diff
@@ -5,13 +5,20 @@
 
 public final class M2SysBiometricsConstants {
 
-    public static final String M2SYS_SERVER_URL = "m2sys-biometrics.server.url";
+    public static final String M2SYS_CLOUD_SCANNER_URL = "m2sys-biometrics.server.url";
 
     public static final String M2SYS_LOCAL_SERVICE_URL = "m2sys-biometrics.local-service.url";
 
-    public static final String M2SYS_USER = "m2sys-biometrics.server.user";
+    public static final String M2SYS_NATIONAL_SERVICE_URL = "m2sys-biometrics.national-service.url";
 
-    public static final String M2SYS_PASSWORD = "m2sys-biometrics.server.password";
+    public static final String M2SYS_LOCAL_USERNAME = "m2sys-biometrics.server.user";
```

Comment:
These are the CloudScanner credentials - if you are renaming, rename to M2SYS_CLOUD_SCANNER_PASSWORD
and M2SYS_CLOUD_SCANNER_USER

Reply:
Ok, M2SYS_CLOUD_SCANNER_USER or M2SYS_CLOUD_SCANNER_USERNAME?
I believe M2SYS_CLOUD_SCANNER_USERNAME is more intuitive.

Reply:
username can stay